src/utils.ts

```diff
@@ -39,3 +41,39 @@ export const verifyToken = (token: string) => {
     }
   }
 };
+
+export const dateFormatter = (date: Date) => {
+  const day = date.getDate();
+  const month = date.getMonth() + 1;
+  const year = date.getFullYear();
+
+  return `${day}-${month}-${year}`;
+};
+
+export const addUsersToDb = async (quantity: number): Promise<User[]> => {
+  const seedPassword = '123456a';
+
+  const users = [];
+
+  for (let i = 0; i < quantity; i++) {
+    const user = new User();
+
+    const { salt, hashedPassword } = generateHash(seedPassword);
+
+    user.name = faker.name.findName();
+    user.email = faker.internet.email();
+    user.birthDate = dateFormatter(faker.date.past());
+    user.password = hashedPassword;
+    user.salt = salt;
+
+    try {
+      users.push(user);
+    } catch (error) {
+      console.log(error);
+    }
+  }
+
+  await AppDataSource.manager.save(users);
+
+  return users;
```

Comment:
da pra retornar direto o resultado do save e jogar fora o async/await
```suggestion
  return AppDataSource.manager.save(users);
```